packages/api/src/beacon/routes/beacon/block.ts

```diff
@@ -246,11 +256,12 @@ export const routesData: RoutesData<Api> = {
 
 /* eslint-disable @typescript-eslint/naming-convention */
 
+type GetBlockReq = {params: {block_id: string}; headers: {accept?: string}};
```

Comment:
Out of curiosity why do we use the snake case for url params.  Seems like a good pattern and I see that its the norm throughout the codebase.  Just want to expand my understanding.

Reply:
its part of the beacon api spec
https://github.com/ethereum/beacon-apis

They say: use this specific url path with these specific url query params, should return a response with these codes and these payloads